plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -132,17 +173,40 @@ When a user deactivates or uninstalls your plugin, it’s good practice to clean
 
 In most cases, plugin data is removed during uninstallation because it indicates the user no longer needs the plugin. As you learned in Module 1, you can use the `register_uninstall_hook()` function to do this.
 
-You can execute the SQL query to delete the table by using the `$wpdb` global variable, which we'll learn more about in the next lesson.
+You can execute the SQL query to delete the table by using the `$wpdb` global variable, which you'll learn more about in a future lesson.
 
 ```php
-register_uninstall_hook( __FILE__, 'my_plugin_delete_table' );
-function my_plugin_delete_table() {
+register_uninstall_hook( __FILE__, 'wp_learn_custom_table_delete_submissions_table' );
+function wp_learn_custom_table_delete_submissions_table() {
 	global $wpdb;
-	$wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}my_plugin_table" );
-	delete_option( 'my_plugin_db_version' );
+	$wpdb->query( "DROP TABLE IF EXISTS {$wpdb->prefix}submissions" );
+	delete_option( 'wp_learn_custom_table_db_version' );
 }
 ```
 
+## A note on custom table names
```

Comment:
This isn't a useful heading for a user to navigate/remember. Can we say something more accurate like "Avoiding Naming Collisions for Custom Tables"?